unable to install kubescape-cli

brew install kubescape-cli
fails on macOS 15.5:

==> Installing kubescape-cli from kubescape/tap
==> make libgit2
Last 15 lines from /Users/leonid/Library/Logs/Homebrew/kubescape-cli/01.make:
   26 | #define NULL ((void*)0)
      |                      ^
/Library/Developer/CommandLineTools/SDKs/MacOSX15.sdk/usr/include/_stdio.h:318:7: note: to match this '('
/private/tmp/kubescape-cli-20250606-7640-3k0iwi/kubescape-3.0.3/git2go/vendor/libgit2/deps/zlib/zutil.h:140:33: note: expanded from macro 'fdopen'
  140 | #        define fdopen(fd,mode) NULL /* No fdopen() */
      |                                 ^
/Library/Developer/CommandLineTools/usr/lib/clang/17/include/__stddef_null.h:26:14: note: expanded from macro 'NULL'
   26 | #define NULL ((void*)0)
      |              ^
3 errors generated.
make[4]: *** [deps/zlib/CMakeFiles/zlib.dir/zutil.c.o] Error 1
make[3]: *** [deps/zlib/CMakeFiles/zlib.dir/all] Error 2
make[2]: *** [all] Error 2
make[1]: *** [static-build/install/lib/libgit2.a] Error 2
make: *** [libgit2] Error 2

If reporting this issue please do so at (not Homebrew/* repositories):
  https://github.com/kubescape/homebrew-tap/issues

--- a/kubescape-cli.rb
+++ b/kubescape-cli.rb
@@ -1,30 +1,15 @@
 class KubescapeCli < Formula
   desc "Kubernetes misconfiguration testing"
   homepage "https://github.com/kubescape/kubescape"
-  url "https://github.com/kubescape/kubescape/archive/v3.0.3.tar.gz"
-  sha256 "b58712f9ef8c270db9f8772f3cadbd5dcb6ee881850379e20f0730c38ce771f6"
+  url "https://github.com/kubescape/kubescape/archive/v3.0.34.tar.gz"
+  sha256 "1da5a3a7b1ef8f38569ee6277fa8cf9d747f9fe379c9283011c7464b7c1003de"
   license "Apache-2.0"
   head "https://github.com/kubescape/kubescape.git", branch: "master"
 
-  depends_on "cmake" => :build
   depends_on "go" => :build
-  depends_on "pkg-config" => :build
-
-  resource "git2go" do
-    url "https://github.com/libgit2/git2go/archive/refs/tags/v33.0.9.tar.gz"
-    sha256 "bcdaa5ed86d7ad513f51cdd80006a23a7fa9d9e68db06b3ce39a25a4196e4d67"
-  end
-
-  resource "libgit2" do
-    url "https://github.com/libgit2/libgit2/archive/refs/tags/v1.3.0.tar.gz"
-    sha256 "192eeff84596ff09efb6b01835a066f2df7cd7985e0991c79595688e6b36444e"
-  end
 
   def install
-    resource("git2go").stage(buildpath/"git2go")
-    resource("libgit2").stage(buildpath/"git2go/vendor/libgit2")
 
-    ENV["CGO_ENABLED"] = "1"
     ENV["GOCACHE"] = buildpath/"cache"
 
     ldflags = %W[
@@ -32,7 +17,6 @@ class KubescapeCli < Formula
       -X github.com/kubescape/kubescape/v3/core/cautils.BuildNumber=v#{version}
     ]
     
-    system "make", "libgit2"
     system "go", "build", *std_go_args(ldflags: ldflags), "-tags", "static,gitenabled", "-o", bin/"kubescape"
 
     output = Utils.safe_popen_read(bin/"kubescape", "completion", "bash")

